vcpkg.json

```diff
@@ -8,34 +8,33 @@
     },
     "sdl2",
     "luafilesystem",
-    "lpeg"
-  ],
-  "features": {
-    "audio": {
-      "description": "In game audio support",
-      "dependencies": [
-         {
-           "name": "fluidsynth",
-           "platform": "!osx",
-           "features": [
-             "sndfile"
-           ]
-        },
+    "lpeg",
+    {
+       "name": "fluidsynth",
+       "platform": "!osx",
+       "features": [
+         "sndfile"
+       ]
+    },
+    {
+      "name": "sdl2-mixer",
+      "features": [
+        "libmodplug",
         {
-          "name": "sdl2-mixer",
-          "features": [
-            "libmodplug",
-            {
-              "name": "fluidsynth",
-              "platform": "!osx"
-            },
-            "libflac",
-            "mpg123",
-            "opusfile"
-          ]
-        }
+          "name": "fluidsynth",
+          "platform": "!osx"
+        },
+        "libflac",
+        "mpg123",
+        "opusfile"
       ]
     },
+    {
+      "name": "freetype",
+      "default-features": false
```

Comment:
Can you leave this out of this PR so it's just dropping the option; we can explore minimizing features later.